dp/configs/forward_config.yaml

```diff
@@ -1,45 +1,49 @@
 
 paths:
-  checkpoint_dir: checkpoints # directory to store model checkpoints and tensorboard
-  data_dir: datasets # directory to store processed data
+  checkpoint_dir: checkpoints   # Directory to store model checkpoints and tensorboard, will be created if not existing.
+  data_dir: datasets            # Directory to store processed data, will be created if not existing.
 
 preprocessing:
-  languages: ['de', 'en_us']
+  languages: ['de', 'en_us']    # All languages in the dataset.
+
+  # Text (grapheme) and phoneme symbols, either provide a string or list of strings.
+  # Symbols in the dataset will be filtered according to these lists!
   text_symbols: 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZäöüÄÖÜß'
   phoneme_symbols: ['a', 'b', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'æ', 'ç', 'ð', 'ø', 'ŋ', 'œ', 'ɐ', 'ɑ', 'ɔ', 'ə', 'ɛ', 'ɝ', 'ɹ', 'ɡ', 'ɪ', 'ʁ', 'ʃ', 'ʊ', 'ʌ', 'ʏ', 'ʒ', 'ʔ', 'ˈ', 'ˌ', 'ː', '̃', '̍', '̥', '̩', '̯', '͡', 'θ']
-  char_repeats: 3 # repeating chars of input text to enable mapping to longer phoneme sequences
-  lowercase: true
-  n_val: 5000
 
-model:
-  type: 'transformer'  # choices: ['transformer', 'lstm', 'autoreg_transformer']
+  char_repeats: 3                # Number of grapheme character repeats to allow for mapping to longer phoneme sequences.
+                                 # Set to 1 for autoreg_transformer.
+  lowercase: true                # Whether to lowercase the grapheme input.
+  n_val: 5000                    # Default number of validation data points if no explicit validation data is provided.
 
-  # transformer params
+
+model:
+  type: 'transformer'            # Whether to use a forward transformer or autoregressive transformer model.
```

Comment:
Since this is config for forward model, the only acceptable value for type should be 'transformer'. The comment seems to suggest that either 'autoreg_transformer' or 'transformer' could be used here.

Reply:
Generally the config should allow for both, the two different examples have just the respective configuration.